### PR TITLE
docs:updated note label syntax in *.md files

### LINKED ISSUE #10458: There are more places in other docs too where this is note is not updated...
There are more places in other docs too where this is not updated...
Maybe we can open an issue for it.

_Originally posted by @parth-gr in https://github.com/rook/rook/pull/10234#discussion_r898109037_

### PR DESCRIPTION
**Pull request by YashasviChaurasia**
The note label is updated in several places in the docs for improved rendering.
Resolves #10458 
Signed-off-by: Yashasvi Chaurasia <yashasvi12977@gmail.com>

### CODE REVIEW

**Review by subhamkrai — CHANGES_REQUESTED** [review]
commitlint CI is failing. Thanks

**Comment by YashasviChaurasia** [comment]
> commitlint CI is failing. Thanks

I have committed the message with commit lint standards can you please help me out why is this check failing.
Thank you

**Comment by subhamkrai** [comment]
> > commitlint CI is failing. Thanks
> 
> I have committed the message with commit lint standards can you please help me out why is this check failing. Thank you

try giving space `docs: updated...`

**Comment by subhamkrai** [comment]
@YashasviChaurasia you need to use `--amend`.

**Comment by subhamkrai** [comment]
also, squash your commits in one commit

File: design/ceph/ceph-config-updates.md [hunk 1]
```diff
@@ -135,8 +135,9 @@ spec:
       debug_osd: 10
    # ...
 ```
-**Note on the above:** all values under config are reported to Ceph as strings, but the yaml should
-support integer values as well if at all possible
+!!!note on the above
```
Comment (subhamkrai) [inline design/ceph/ceph-config-updates.md:138]:
```suggestion
!!! note 
```

File: design/ceph/ceph-config-updates.md [hunk 2]
```diff
@@ -135,8 +135,9 @@ spec:
...
+    All values under config are reported to Ceph as strings, but the yaml should
+    support integer values as well if at all possible
```
Comment (subhamkrai) [inline design/ceph/ceph-config-updates.md:140]:
```suggestion
    support integer values as well if at all possible.
```

File: design/ceph/interacting-with-rook-resources.md [hunk 3]
```diff
@@ -38,6 +38,7 @@ Another example on CephFilesystem with ID=a:
 	app.kubernetes.io/managed-by : "rook-ceph-operator"
 	app.kubernetes.io/created-by : "rook-ceph-operator"
 	rook.io/operator-namespace   : "rook-ceph"
-``` 
+```
 
-**NOTE** : A totally unique string for an application can be built up from (a) app.kubernetes.io/component, (b) app.kubernetes.io/part-of, (c) the resource's namespace, (d) app.kubernetes.io/name, and (e) app.kubernetes.io/instance fields. For the example above, we could join those fields with underscore connectors like this: cephclusters.ceph.rook.io_rook-ceph_rook-ceph_ceph-mon_a. Note that this full spec can easily exceed the 64-character limit imposed on Kubernetes labels.
\ No newline at end of file
+!!!note
```
Comment (subhamkrai) [inline design/ceph/interacting-with-rook-resources.md:43]:
```suggestion
!!! note
```

**Review by subhamkrai — CHANGES_REQUESTED** [review]

**Review by subhamkrai — CHANGES_REQUESTED** [review]
@YashasviChaurasia please rebase on master. you have other commits too.

File: pkg/operator/ceph/csi/spec.go [hunk 4]
```diff
@@ -39,6 +39,7 @@ import (
 )
 
 type Param struct {
+
```
Comment (Sheetalpamecha) [inline pkg/operator/ceph/csi/spec.go:42]: Can avoid extra line

File: Documentation/Contributing/development-flow.md [hunk 5]
```diff
@@ -287,9 +287,10 @@ Signed-off-by: First Name Last Name <email address>
 
 The `component` **MUST** be in the [list checked by the CI](https://github.com/rook/rook/blob/master/.commitlintrc.json).
 
-Note: sometimes you will feel like there is not so much to say, for instance if you are fixing a typo in a text.
-In that case, it is acceptable to shorten the commit message.
-Also, you don't always need to close an issue, again for a very small fix.
+note!!!
```
Comment (Sheetalpamecha) [inline Documentation/Contributing/development-flow.md:290]: The syntax should be corrected to  - !!! Note

**Review by Sheetalpamecha — CHANGES_REQUESTED** [review]

**Comment by parth-gr** [comment]
@YashasviChaurasia any update??

I see you might be stuck at squashing,

Can you do 
`git remote add upstream https://github.com/rook/rook.git`
`git fetch --all`
`git rebase upstream/master`

**Comment by YashasviChaurasia** [comment]
> @YashasviChaurasia any update??
> 
> I see you might be stuck at squashing,
> 
> Can you do `git remote add upstream https://github.com/rook/rook.git` `git fetch --all` `git rebase upstream/master`

The problem I am facing is kind of absurd as whenever I squash and submit a PR that is squashed nicely, a new change is requested, and when I modify the change and squash again then, the checks fail, and I have to resubmit multiple times to get the checks to work which further creates multiple PRs, and I need to squash them all again ... and hence this loop never seem to end.

**Comment by YashasviChaurasia** [comment]
> > @YashasviChaurasia any update??
> > I see you might be stuck at squashing,
> > Can you do `git remote add upstream https://github.com/rook/rook.git` `git fetch --all` `git rebase upstream/master`
> 
> The problem I am facing is kind of absurd as whenever I squash and submit a PR that is squashed nicely, a new change is requested, and when I modify the change and squash again then, the checks fail, and I have to resubmit multiple times to get the checks to work which further creates multiple PRs, and I need to squash them all again ... and hence this loop never seem to end.

I was busy for a while, will try to work and get this done today.
The problem is not with the PR but with the checks, and to re-run the checks, I need to change the PR a little, which is already complete, which causes issues while submitting GitHub PR in a proper format.

**Comment by YashasviChaurasia** [comment]
```Canary integration tests / multi-cluster-mirroring (pull_request)```
@parth-gr I am unable to understand why this check is failing.
Kindly help me out with this.
Thank you

**Comment by YashasviChaurasia** [comment]
@parth-gr the rebase if failing and I am not able to switch branches now on the terminal.

**Comment by BlaineEXE** [comment]
@YashasviChaurasia please rebase your branch on the latest upstream rook master. This pull request is attempting to merge 142 commits that are mostly duplicates of already-existing changes. We simply cannot merge such a PR. If you aren't sure how to rebase on the latest upstream master, please look to the plethora of helpful articles written about it like this one: https://medium.com/@topspinj/how-to-git-rebase-into-a-forked-repo-c9f05e821c8a.

**Comment by YashasviChaurasia** [comment]
@BlaineEXE I completely understand your point, and I am ready to rebase, but I am unable to do it on my local repo due to some issues I have created.
Can I delete this PR and create a fresh PR, as I am unable to switch branches on my local repo?
Thank you

**Comment by BlaineEXE** [comment]
> @BlaineEXE I completely understand your point, and I am ready to rebase, but I am unable to do it on my local repo due to some issues I have created. Can I delete this PR and create a fresh PR, as I am unable to switch branches on my local repo? Thank you

You're free to resolve it in any way you need to. For the health of the Rook repository, we cannot merge this PR in its current state. If you create a new PR, it would be most helpful to link to the PR taking over for this one.

**Comment by YashasviChaurasia** [comment]
@BlaineEXE 
I am closing this issue, and the link to a new issue has been linked below.
I have ensured all the checks pass and only make a clean single commit.
Thank you
New PR: rook/rook#11171

**Pull request closed without merging**